.github/workflows/cleanup.yml

```
@@ -28,18 +28,19 @@ env:
   GAR_REPO: 'us-docker.pkg.dev/github-metrics-ci/ci-images'
   INTEGRATION_PROJECT_ID: 'github-metrics-aggreg-i-64d426'
   INTEGRATION_REGION: 'us-central1'
-  INTEGRATION_WEBHOOK_SERVICE_NAME: 'github-metrics-webhook-2578'
-  INTEGRATION_RETRY_SERVICE_NAME: 'github-metrics-retry-c415'
 
 permissions:
   contents: 'read'
   id-token: 'write'
 
 jobs:
-  # cleanup_webhook_cloudrun_revisions deletes all Webhook Cloud Run revisions for the given service
-  # that are more than 5 hours old
-  cleanup_webhook_cloudrun_revisions:
+  # cleanup_cloudrun_revisions deletes all Webhook and Retru Cloud Run revisions
+  # for the given service that are more than 5 hours old.
+  cleanup_cloudrun_revisions:
     runs-on: 'ubuntu-latest'
+    strategy:
+      matrix:
+        service_name: ['github-metrics-webhook-2578', 'github-metrics-retry-c415']
```

nit: prefer long form

```suggestion
        service_name:
          - 'github-metrics-webhook-2578'
          - 'github-metrics-retry-c415'
```